### ISSUE TITLE
How to add a item programatically?

### ISSUE THREAD

**Issue by pietrofxq** [tamtakoe/oi.select#105]
I need to pass a param to the page, like /url/id?filter=dogs and then redirect to that page and make the filter "dogs" already selected on page load. Is that possible?

**Comment by AlayVora** [comment]
Yes, it is possible. When you redirect to another page, you just need to store the filter value in some variable and then set the oi-select to have a default value.

In the following example [link](https://tamtakoe.github.io/oi.select/#/select/#multiplelimit), you can see that bundle variable already contains some values and which are by default shown on the oi-select.

**Comment by pietrofxq** [comment]
I know you can have default values, but it's not possible to add items later. If you push a item to the array model, the select plugin doesn't update.

**Comment by NicoJuicy** [comment]
When you set a new object to the model. The plugin detects the changes.

create a new variable, which contains the ng-model array
push to that variable
set ng-model to that variable

I think that should work
